Import resolution is not working

### What happened?

I can not get pyright to work with my virtual environment which is set as the project interpreter in **PyCharm Professional 2024.3.1.1** on **Windows 10**. It shows missing imports, even though the modules source paths are added to the project.

1. Special note about my setup is, I have multiple namespace packages ( i.e multiple project roots ) in my pycharm project.
2. The Virtual Environment even have a .pth file in the site-packages that points to additional python packages
3. I do not have any system level python interpreter exposed

![Image](https://github.com/user-attachments/assets/6b8ecffe-90b2-4843-9494-c74c3e60f29c)

### Steps to reproduce

1. In Pycharm, Create a new project and use a virtual environment created by uv for Python 3.10
2. Install pyright using uv `uv tool install pyright[nodejs]`
3. Configure PyCharm Pro's pyright plugin to use the binaries installed by uv `%UserProfile%/.local/bin`
4. Enable pyright inspection
5. Everything works, except pyright reports `reportMissingImports` for all my internal modules as well as modules that are pip installed


### Relevant log output or stack trace

```text
[Trace - 23:55:12] Sending request 'textDocument/codeAction - (241)'.
Params: {
  "textDocument": {
    "uri": "file:///D:/test/TestProject/package1/pyright_playground.py"
  },
  "range": {
    "start": {
      "line": 0,
      "character": 5
    },
    "end": {
      "line": 0,
      "character": 8
    }
  },
  "context": {
    "diagnostics": [
      {
        "range": {
          "start": {
            "line": 0,
            "character": 5
          },
          "end": {
            "line": 0,
            "character": 8
          }
        },
        "severity": 1,
        "code": "reportMissingImports",
        "codeDescription": {
          "href": "https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportMissingImports"
        },
        "source": "Pyright",
        "message": "Import \"pak\" could not be resolved"
      }
    ],
    "only": [
      "quickfix"
    ],
    "triggerKind": 2
  }
}

---------
Logs
----------
Server root directory: file:///c%3A/.conda/envs/py310/Lib/site-packages/pyright/dist/dist
Pyright language server 1.1.392 starting
Starting service instance "package2"
Starting service instance "TestProject"
Auto-excluding **/.*
Auto-excluding **/node_modules
Auto-excluding **/__pycache__
No include entries specified; assuming e:\package2
Found 2 source files
Auto-excluding **/.*
Auto-excluding **/__pycache__
Auto-excluding **/node_modules
No include entries specified; assuming d:\test\TestProject
Found 2 source files
```

### Operating system

Windows

Additional stack trace, on a bigger project.

`java.lang.Exception: Timeout error while shutdown the language server 'com.insyncwithfoo.pyright'
	at com.redhat.devtools.lsp4ij.LanguageServerWrapper.shutdownLanguageServerInstance(LanguageServerWrapper.java:586)
	at com.redhat.devtools.lsp4ij.LanguageServerWrapper.shutdownAll(LanguageServerWrapper.java:552)
	at com.redhat.devtools.lsp4ij.LanguageServerWrapper.lambda$stop$15(LanguageServerWrapper.java:520)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1491)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2073)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2035)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
	at com.redhat.devtools.lsp4ij.LanguageServerWrapper.shutdownLanguageServerInstance(LanguageServerWrapper.java:580)
	... 9 more`

--- a/src/main/kotlin/com/insyncwithfoo/pyright/lsp4ij/PyrightServerClient.kt
+++ b/src/main/kotlin/com/insyncwithfoo/pyright/lsp4ij/PyrightServerClient.kt
@@ -16,7 +16,7 @@ internal class PyrightServerClient(project: Project) : LanguageClientImpl(projec
         val settings = project.createLSPSettingsObject()
         
         return when (section) {
-            "python" -> settings
+            "python" -> settings.python
             "python.analysis" -> settings.python.analysis
             "pyright" -> settings.pyright
             else -> null

